rinja_parser/src/lib.rs

```diff
@@ -416,7 +416,41 @@ fn separated_digits(radix: u32, start: bool) -> impl Fn(&str) -> ParseResult<'_>
     }
 }
 
-fn str_lit(i: &str) -> ParseResult<'_> {
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum StrPrefix {
+    None,
```

Comment:
I think the code would be more readable `StrLit` used `Option<StrPrefix>`, and this variant was removed.

Reply:
I originally did that. Didn't like it for some reason but I think it's more logical to do as you suggest. :+1: